AEFeinstein/Super-2024-Swadge-FW
# Audio dac

### Description

Added support for the speaker in both hardware and the emulator.

Note, two git submodules were added in `emulator/src-lib/`.

### Test Instructions

* Run on actual hardware with Speaker SAO
    * Play a variety of songs in the jukebox using the speaker, headphones
    * Check volume dial
* Run emulator
    * Play a variety of songs in the jukebox
    * Test mic input in Colorchord

### Ticket Links

n/a

### Readiness Checklist
